pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+If you are interested in assuming ownership of a PHEP, you can also do this via the pull request discussion thread. Mention both the original author and `@heliophysicsPy/phep-editors` in a comment on the pull request. (If the original author's GitHub username is unknown, use email.) If the original author doesn't respond in a timely manner, the PHEP editors will make a unilateral decision, which can be reversed.
+
+# PHEP Editor Responsibilities & Workflow
+<a name="phep-editor-responsibilities-workflow"></a>
+A PHEP editor must be added to the `@heliophysicsPy/pep-editors` group on GitHub and must watch the `standards` repository.
+
+Other community members with write access to the `standards` repository may handle the tasks that would normally be taken care of by the PHEP editors. Alternately, even those with write access may request assistance from PHEP editors by mentioning `@heliophysicsPy/phep-editors` on GitHub.
+
+For each new PHEP that comes in an editor does the following:
+
+* Read the PHEP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don't seem likely to be accepted.
+* The title should accurately describe the content.
+* The file name extension is correct (i.e. `.md`).
+* Ensure that everyone listed as a co-author of the PHEP who has write access to the PHEP repository is added to [.github/CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).
+* Skim the PHEP for obvious defects in language (spelling, grammar, sentence structure, etc.). Editors may correct problems themselves, but are not required to do so.
+* If a project is portrayed as benefiting from or supporting the PHEP, make sure there is some direct indication from the project included to make the support clear. This is to avoid a PHEP accidentally portraying a project as supporting a PHEP when in fact the support is based on conjecture.
+
+If the PHEP isn't ready, an editor will send it back to the author for revision, with specific instructions. If Markdown formatting is a problem, ask the author(s) to resubmit.
+
+Once the PHEP is ready for discussion, a PHEP editor will:
+
+* Assign a PHEP number (almost always just the next available number, generally avoiding special/joke numbers).
+* Check that the author has correctly labeled the PHEP's type ("Standards Track", "Informational", or "Process"), and marked its status as "Draft".
+* Mark the PR ready for review.
+* Inform the author of the next steps (update the PHEP with the PR link, post an announcement, etc).
+
+Updates to existing PHEPs should be submitted as comments in the GitHub pull request.
+
+PHEP editors don't pass judgment on PHEPs. They merely do the administrative & editorial part (which is generally a low volume task). They may participate in the community discussion of a PHEP but have no obligation to be advocates.
```

Comment:
Suggestion for clarity: `PHEP editors do not make judgments on the content or merits of PHEPs.`